test/object-store/sync/flx_sync.cpp

```diff
@@ -3062,10 +3217,7 @@ TEST_CASE("flx: data ingest - dev mode", "[sync][flx][data ingest][baas]") {
 
     harness.do_with_new_realm(
         [&](SharedRealm realm) {
-            auto table = realm->read_group().get_table("class_TopLevel");
-            auto new_query = realm->get_latest_subscription_set().make_mutable_copy();
-            new_query.insert_or_assign(Query(table));
-            std::move(new_query).commit();
+            subscribe_to_all(*realm);
```

Comment:
this should not be needed at all. I think it was added due to a bug in the server requiring some subscriptions. Can you check if we can remove it?

Reply:
Can we check below the objects are created on the server?